math/libfaiss/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libfaiss
+version             20180214
+categories          math
+license             BSD
+platforms           darwin
+maintainers         fb.com:hoss \
+                    fb.com:matthijs
+description         Efficient similarity search library from Facebook AI Research.
+long_description    Library for efficient similarity search and clustering of dense vectors.
+homepage            https://github.com/facebookresearch/faiss
+fetch.type          git
+git.url             https://github.com/facebookresearch/faiss.git
+git.branch          56383610bcb982d6591e2e2bea3516cb7723e04a
```

Comment:
If this is coming from github, please use the github portgroup which automates most of these sorts of fetch and other details.